src/bin/index.js

```diff
@@ -124,26 +138,41 @@ yargs(hideBin(process.argv))
           'Respond `yes` to all confirmation prompts.\nAllows running non-interactively within a script.',
       },
     },
-    async (argv) => await deploy(argv)
-  )
-  .command(
-    ['example [name]', 'e [name]'],
-    'Create an example project',
-    {
+    handler: async (argv) => await deploy(argv),
+  };
+}
+
+function exampleCli() {
+  return {
+    command: ['example [name]', 'e [name]'],
+    describe: 'Create an example project',
```

Comment:
```suggestion
    describe: 'Create an example project with test files',
```

Reply:
Why do you think we need to emphasise that test files will be created for the example project as well?